src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -554,5 +554,12 @@ static String getResource(String key) {
                     "MSI Token failure: Failed to acquire access token from IMDS, unexpected error occurred."},
             {"R_MSITokenFailureEndpoint", "MSI Token failure: Failed to acquire token from MSI Endpoint"},
             {"R_propertyNotSupported",
-                    "Microsoft JDBC Driver for SQL Server currently does not support the property: {0}"}};
+                    "Microsoft JDBC Driver for SQL Server currently does not support the property: {0}"},
+            {"R_useFmtOnlyPropertyDescription",
+                    "Determines whether to parse user SQL and execute a FMTONLY query to retrieve parametermetadata."},
```

Comment:
hmm should this be either "parameter metadata" or, "ParameterMetaData"?

Reply:
```suggestion
                    "Determines whether to enable/disable use of SET FMTONLY to retrieve parameter metadata."},
```